en/flight_modes/offboard.md

```diff
@@ -30,16 +30,21 @@ The action is defined in the parameters [COM_OBL_ACT](#COM_OBL_ACT) and [COM_OBL
 
 ### Copter/VTOL
 
-* [SET_POSITION_TARGET_LOCAL_NED](https://mavlink.io/en/messages/common.html#SET_POSITION_TARGET_LOCAL_NED) - Control vehicle position, velocity, or thrust. 
-  - Acceleration setpoints are combined to create a 'thrust' setpoint.
+* [SET_POSITION_TARGET_LOCAL_NED](https://mavlink.io/en/messages/common.html#SET_POSITION_TARGET_LOCAL_NED)
+  * The following input combinations are supported: <!-- https://github.com/PX4/Firmware/blob/master/src/lib/FlightTasks/tasks/Offboard/FlightTaskOffboard.cpp#L166-L170 -->
+    * Position setpoint (only)
+    * Velocity setpoint (only)
+    * *Thrust* setpoint (only)
+      > **Note** Acceleration setpoint values are mapped to create a normalized thrust setpoint (i.e. acceleration setpoints are no "properly" supported).
+    * Position setpoint **and** velocity setpoint (velocity used as feedforward)

```

Comment:
What does "velocity used as feedforward" mean?

Reply:
it means that the given feedforward-velocity setpoint `v_sp` is added to the output of the position P-controller:
`P(p_sp) + v_sp = v_sp`  where `P` is a function representing a position-controller (in the case of PX4, it is just a gain of the position setpoint and the position estimation)

Reply:
Maybe call differently v_sp at left and at the right of "=" because they are two different things ;) . The one of the left is the setpoint given by the "user". The one of the right is the output of the controller which is the reference for the velocity controller.

Reply:
Thanks for the clarification. I changed this to 

>    Position setpoint **and** velocity setpoint (the velocity setpoint is used as feedforward; it is added to the output of the position controller and the result is used as the input to the velocity controller).

I'm going to merge now.